keras_cv/layers/object_detection/multi_class_non_max_suppression.py

```diff
@@ -48,6 +50,7 @@ def __init__(
         confidence_threshold=0.5,
         max_detections=100,
         max_detections_per_class=100,
+        output_ragged=True,
```

Comment:
IIRC we had discussed this API in the past and decided it wasn't worthwhile -- is there a reason this can't always be true?

Conversely, can it always be false? It seems like since we provide a `to_ragged` construct, we can direct users to that if they want to prune results from a model?

Reply:
We can just make it true.

Reply:
> Conversely, can it always be false? It seems like since we provide a to_ragged construct, we can direct users to that if they want to prune results from a model?

This is more just confusing to users - we have seen this in the issues already.  "Why does my model output all -1s". Its a lot clearer if it outputs no boxes that it just didn't make any detections.